src/components/LayersDrawer/LayersDrawer.tsx

```
@@ -5,9 +5,12 @@ import { useTranslation } from 'react-i18next'
 import StyledSwipeableDrawer from '../StyledSwipeableDrawer/StyledSwipeableDrawer'
 import StyledIconButtonWithTooltip from '../StyledIconButtonWithTooltip/StyledIconButtonWithTooltip'
 import styles from './LayersDrawer.module.scss'
-import { LayerInfo, parentLayerTitles } from '../../data/mapData'
+import { atlasBenthicColors, atlasBenthicLayers, parentLayerTitles } from '../../data/mapData'
```

ESLint

[error] 'atlasBenthicColors' is defined but never used.

(@typescript-eslint/no-unused-vars)